gulpfile.js

```diff
@@ -60,7 +60,8 @@ var gulp = require('gulp'),
             singleRun: true
         },
         sass: {
-            includePaths: bourbon.includePaths
+            includePaths: bourbon.includePaths,
+            sourceComments: true
```

Comment:
Yeah-- we're not incredibly concerned about size of static assets and considering the inconsistencies with sourcemap support in chrome this is the next best option.  If chrome starts displaying sourcemaps, then we can turn this off.  (I can't find a reason why they don't work; they work in firefox just fine).